libcxx/test/libcxx/containers/sequences/vector.bool/abi.compile.pass.cpp

```diff
@@ -6,6 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+// REQUIRES: libcpp-abi-version=1
```

Comment:
```suggestion
// UNSUPPORTED: libcpp-abi-has-no-compressed-pair-padding
```

And then introduce `libcpp-abi-has-no-compressed-pair-padding` in `features.py` like we do for other ABI macros. We should probably get rid of the `libcpp-abi-version` Lit feature altogether since we do pretty much everything based on the individual ABI macros nowadays.